README.md

```diff
@@ -8,8 +8,89 @@ User documentation for the SecureDrop Workstation.
+
+### Set up pyenv and pyenv-virtualenv
+
+From the Terminal, run
+`brew update && brew upgrade && brew install pyenv pyenv-virtualenv`.
+
+You may want to read more about [pyenv](https://github.com/pyenv/pyenv) and
+[pyenv-virtualenv](https://github.com/pyenv/pyenv-virtualenv).
+
+To ensure that these tools run properly, add the following lines to your
+`~/.bash_profile` (or `~/.zprofile`) by opening that file in your favourite
+text editor (you can use `nano ~/.bash_profile` to work on the command-line):
+
+```
+eval "$(pyenv init -)"
+eval "$(pyenv virtualenv-init -)"
+export LC_ALL=en_US.UTF-8
+export LANG=en_US.UTF-8
+```
+
+### Fix brew errors
+
+If you run `brew doctor`, you will see the following warning:
+
+```
+Warning: "config" scripts exist outside your system or Homebrew directories.
+`./configure` scripts often look for *-config scripts to determine if
+software packages are installed, and what additional flags to use when
+compiling and linking.
+
+Having additional scripts in your path can confuse software installed via
+Homebrew if the config script overrides a system or Homebrew provided
+script of the same name. We found the following "config" scripts:
+
+```
+
+You can ignore this warning, or use one of the two possible fixes documented
+in the [pyenv Github repo](https://github.com/pyenv/pyenv/issues/106).
+
+For example, creating a file called `~/bin/brew` with the following contents:
+
+```
+#!/bin/sh
+
+# from https://github.com/pyenv/pyenv/issues/106
+if which pyenv > /dev/null 2>&1; then
+  # assumes default location of brew in `/usr/local/bin/brew`
+  /usr/bin/env PATH="${PATH//$(pyenv root)\/shims:/}" /usr/local/bin/brew "$@"
+else
+  /usr/local/bin/brew "$@"
+fi
+```
+and adding `~/bin` to your PATH variable (`EXPORT PATH=${HOME}/bin:${PATH}`)
+is a workaround for this issue.
+
+Once you have finished making these changes, reload your shell by typing `source ~/.bash_profile` in the Terminal app.
+
+## Install appropriate version of Python
+
+In the terminal, install Python 3.5.9 with `pyenv install 3.5.9`
+
+## Clone repo, set up environment
+
+- Clone this repo and `cd` into the `securedrop-workstation-docs` directory.
+
+- Type `pyenv local 3.5.9` to tell your system to use this version of Python in
+this project.
+
+- Type `pyenv virtualenv sdwdocs` (you may replace "sdwdocs" with whatever name you wish to give this virtual environment).
+
+- Activate this environment by typing `pyenv activate sdwdocs`. You will see the name in parentheses to indicate that it is active.
+
+- Install required dependencies via `pip install --require-hashes -r requirements/requirements.txt`.
+
+When the installation of dependencies completes, you can run `make docs` to build the documentation and view it in your browser at `127.0.0.1:8000`, or `make docs-lint` and `make docs-lintcheck` to check for style/formatting errors.
+
+- **When you're done**: deactivate the virtual environment by typing `pyenv deactivate sdwdocs`.
```

Comment:
Inconsistent formatting, only place bolding is used.

Reply:
fixed in 6d1db2b